docs/dev/replay-exec.md

```diff
@@ -0,0 +1,86 @@
+This doc describes how to dump an Exec runtime meta and column batch data to a directory,
+and replay the Exec with the dumped meta and data. When encountering a perf issue using this tool
+can dump the runtime meta/data and then replay. It will spend less time when collecting NSYS and
+NCU information when replaying the dumped runtime meta/data.
+
+# Dump a Exec meta and a column batch data
+
+## compile Spark-Rapids jar
+e.g.: compile for Spark330
+```
+mvn clean install -DskipTests -pl dist -am -DallowConventionalDistJar=true -Dbuildver=330 
+```
+Note: Should specify `-DallowConventionalDistJar`, this config will make sure to generate a
```

Comment:
I would rather see this first saying you must build with this option and then below that have an example and point to our build docs

Reply:
Done.